versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-commands/install.mdx

```diff
@@ -0,0 +1,36 @@
+# install
+
+Use this command to install the required workloads to run KubeSlice Controller and KubeSlice Worker.
```

Comment:
KubeSlice Worker = worker clusters? 

If yes, then please change it.

Reply:
done